man/systemd.network.xml

```diff
@@ -1618,6 +1618,13 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>MTUBytes=</varname></term>
```

Comment:
hmm, this ultimately sets the MTU for the DHCP *routes*, right? Hence I think we should name this RouteMTUBytes=, no? Similar to RouteMetric= and similar?

Reply:
yes. updated.